src/main/java/com/cqse/teamscaleupload/TeamscaleUpload.java

```diff
@@ -131,7 +132,7 @@ public void validate(ArgumentParser parser) throws ArgumentParserException {
             }
 
             if (hasMoreThanOneCommitOptionSet()) {
-                throw new ArgumentParserException("You used more than one of --detect-commit, --commit and --timestamp." +
+                throw new ArgumentParserException("You used more than one of --detect-commit, --commit and --branch-and-timestamp." +
```

Comment:
As we removed the `detect-commit` option, we should not mention it here anymore, right?